### ISSUE TITLE
Application/json not work with Json in Body

### ISSUE THREAD

**Issue by seceba** [rknell/alfred#100]
Hello, first of all thank you for this beautiful project.

I have been developing my project with Alfred for a while and I am almost halfway there.

However, I am facing a problem that I have not been able to solve for a few days.

For example, when I try to send a json as raw, I get an error;

{"foo": "poo"} when I send it as application/json in header-type I get the error "2022-07-12 13:54:56.425618 - error - Wrong FormatException: Unexpected character (at line 2, character 10)".

However, if I make the header-type text/plain, I can solve this problem, but I still don't understand why I can't send it as application/json.

Likewise, another problem is that if I use quotation marks in the form-data key value, it cannot read it and gives an error. This is not a problem but I can't handle this problem, I can't show a specific error accordingly.

In short, I cannot send a json value as application/json in Body, I get an error, how can I fix this?

Thank you very much
